web/themes/custom/customtheme/customtheme.info.yml

```diff
@@ -9,6 +9,7 @@ dependencies:
 libraries:
   - customtheme/global
   - customtheme/fonts
+  - customtheme/buttons
```

Comment:
Does it make sense to load that library on every page? If it is only for the back to top button, then it should be attached to it.
Even if you right now have that button on every page, it is no effort to isolate the library only to the element that needs it.